.circleci/config.yml

```diff
@@ -175,3 +189,10 @@ workflows:
                 - master
     jobs:
       - tag-release-candidate
+  auto-rebase:
+    jobs:
+      - auto-rebase:
+          filters:
+            branches:
+              only:
+                - master
```

Comment:
Could we add another job and try to rebase when we create PR?
Like
git checkout  jz/rebase-custom1
git checkout -b jz/rebase-custom1-temp
git rebase master
<check that custom branch was "up-to-master" before this PR, otherwise stop
git rebase <PR branch>
<check that we are able to do rebase
git branch -D  jz/rebase-custom1-temp

Reply:
@vitaliyreznikov  The PR version of this is a followup ticket
https://boltpay.atlassian.net/browse/MX-1318

Reply:
Got it. Thanks for explanation Usman.